cmd/lakectl/cmd/dbt.go

```diff
@@ -146,7 +147,7 @@ func getDbtTables(projectRoot string) ([]model, error) {
 	dbtCmd.Dir = projectRoot
 	output, err := dbtCmd.Output()
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to run dbt debug with output:\n %s\n and error: %s", output, err)
```

Comment:
not sure about this one - it looks like an error that goes to the output not something that failed - it capture the output.
is that right?

Reply:
For some reason, when dbt debug returns an error, the information for the error is printed to the standard outpout

Reply:
Changed it, now, in case of error: the output is first printed as output and the error as error